blogs/views.py

```diff
@@ -21,8 +21,11 @@ def home(request):
     if request.user.is_authenticated:
         data = {'user': request.user}
         return render(request, 'feed.html', data)
-    return render(request, 'home.html')
+    return log_in(request, 'home.html') #accomodate login modal in home view
```

Comment:
Could you explain what this change does?

Reply:
instead of just rendering the home view, it also renders a login form which is opened in the home view when the modal button is triggered. So when home.html is rendered, login is also automatically rendered as well